### PR TITLE
Simplify comment about removed markets

### LINKED ISSUE #362: misleading comment
https://github.com/morpho-org/metamorpho/blob/0b63c348445fea57532e3dd5d4962961d770d5a6/src/interfaces/IMetaMorpho.sol#L143

what do we mean here?

### PR DESCRIPTION
**Pull request by QGarchery**
Refactor because it is already explained below the different cases where the market can be removed (also the first comment was missing a case)

fixes #362

### CODE REVIEW

**Review by Jean-Grimal — COMMENTED** [review]
Not sure about this one

File: src/interfaces/IMetaMorpho.sol [hunk 1]
```diff
@@ -137,9 +137,8 @@ interface IMetaMorphoBase {
     /// increase the cost of depositing to the vault.
     function setSupplyQueue(Id[] calldata newSupplyQueue) external;
 
-    /// @notice Sets the withdraw queue as a permutation of the previous one, although markets with both zero cap and
-    /// zero vault's supply can be removed from the permutation.
-    /// @notice This is the only entry point to disable a market.
+    /// @notice Sets the withdraw queue as a permutation of the previous one, although some markets can be removed.
+    /// @notice This is the only entry point to remove a market.
```
Comment (adhusson) [inline src/interfaces/IMetaMorpho.sol:141]:
```suggestion
    /// @notice Updates the withdraw queue. No market can be added, some can be removed (see below).
    /// @notice This is the only entry point to remove a market.
```

Comment (MerlinEgalite) [reply src/interfaces/IMetaMorpho.sol:141]: This version is simple and clear I like it

Comment (Rubilmax) [reply src/interfaces/IMetaMorpho.sol:141]:
```suggestion
    /// @notice Sets the withdraw queue as a permutation of the previous one, although some markets can be removed.
    /// @notice This is the only entry point to remove a market from the withdraw queue.
```

(optional)

**Review by Rubilmax — APPROVED** [review]

File: src/interfaces/IMetaMorpho.sol (hunk 1 above)
Comment (MathisGD) [inline src/interfaces/IMetaMorpho.sol:141]: I still find this comment misleading (#362) even if I understand what it means

Comment (MerlinEgalite) [reply src/interfaces/IMetaMorpho.sol:141]: What's your prop?

Comment (MathisGD) [reply src/interfaces/IMetaMorpho.sol:141]: remove it

Comment (MathisGD) [reply src/interfaces/IMetaMorpho.sol:141]:
```suggestion
```

Comment (MathisGD) [inline src/interfaces/IMetaMorpho.sol:141]:
```suggestion
    /// @notice Updates the withdraw queue. Some markets can be removed, but no market can be added.
```

**Review by Rubilmax — APPROVED** [review]

**Review by MerlinEgalite — APPROVED** [review]

**Review by MathisGD — APPROVED** [review]

**Review by Jean-Grimal — APPROVED** [review]

**Pull request merged**